src/chat/chatter.py

```diff
@@ -369,8 +369,58 @@ def set_color(self):
             color = pcolors.getUserColor(_id, name)
         self.setForeground(QtGui.QColor(color))
 
+    def name_used_by_other(self):
+        api_link = 'https://api.faforever.com/data/player?include=names&' \
+                   'filter=(login==' + self.user.name + ',names.name==' + self.user.name + ')'
+        response = requests.get(api_link).json()
+        if response.get('errors'):
+            return ''
+        result = ''
+        if response.get('data') is not None and len(response['data']) >= 1:
+            for ply in response['data']:
+                if ply['type'] == 'player':
+                    if ply['attributes']['login'] != self.user.name:
+                        result += (ply['attributes']['login'] + '<br/>')
+        if len(result) > 1:
+            result = 'The name has previously been used by :<br/><br/>' + str(result)
+        else:
+            result = 'The name has never been used by anyone else.'
+        return result
+
+    def names_previously_known(self):
+        api_link = 'https://api.faforever.com/data/player?include=names&fields[player]=login,names&' \
+                   'fields[nameRecord]=name,changeTime&&filter[player]=login==' + self.user.name
+        response = requests.get(api_link).json()
+        if response.get('errors'):
+            return ''
+        if response.get('included') is not None and len(response['included']) >= 1:
+            nick_list = 'The player has previously been known as :' \
```

Comment:
I think it would be better if we did this formatting through qt .ui files, and then just filled in widgets.